README.md

```diff
@@ -86,15 +86,8 @@ The datasets that are implemented for this benchmark can be found in `./datasets
 ```
 chapterbreak
 colours
-conflicting_personal_info
-delayed_recall
-how_to_think
-instruction_recall
 jokes
-kv
-locations
 locations_directions
-names
```

Comment:
These datasets might not be 100% reliable and therefore not included in the final configuration, but they are still implemented in the repository.

Reply:
I guess... I can restore it.